tests/models/opt/test_modeling_opt.py

```diff
@@ -428,3 +428,25 @@ def test_generation_post_attn_layer_norm(self):
             predicted_outputs += generated_string
 
         self.assertListEqual(predicted_outputs, EXPECTED_OUTPUTS)
+
+    @require_torch_gpu
```

Comment:
```suggestion
    @require_torch_gpu
    @slow
```

Reply:
I think the decorator is on the class definition so I think all the functions that are from this class will be `slow`

Reply:
ah, I see. I stand corrected. All is good.